controller/customdomain_utils.go

```diff
@@ -104,6 +118,82 @@ func FindCustomDomainCondition(conditions []customdomainv1alpha1.CustomDomainCon
 	return nil
 }
 
+// Take an ingress controller managed by the custom domains operator and release it back to the
+// cluster ingress operator. Also schedule it onto customer worker nodes from the Red Hat managed infra
+// nodes.
+func (r *CustomDomainReconciler) returnIngressToClusterIngressOperator(reqLogger logr.Logger, instance *customdomainv1alpha1.CustomDomain) (ctrl.Result, error) {
+	reqLogger.Info(fmt.Sprintf("Removing operator management labels from %s's underlying ingress controller", instance.Name))
+
+	ingressName := instance.Name
+	customIngress := &operatorv1.IngressController{}
+
+	err := r.Client.Get(context.TODO(), types.NamespacedName{
+		Namespace: ingressOperatorNamespace,
+		Name:      ingressName,
+	}, customIngress)
+	reqLogger.Info(fmt.Sprintf("Fetching ingress controller of name %s", instance.Name))
+
+	customIngress.Labels = map[string]string{}
+	if err != nil {
```

Comment:
Check errors immediately, logging tweak

```suggestion
  customIngress.Labels = map[string]string{}
  reqLogger.Info(fmt.Sprintf("Fetching ingress controller: %s/%s", ingressOperatorNamespace, ingressName))
	err := r.Client.Get(context.TODO(), types.NamespacedName{
		Namespace: ingressOperatorNamespace,
		Name:      ingressName,
	}, customIngress)
	if err != nil {
```